src/Explorer/Explorer.tsx

```diff
@@ -1303,23 +1286,22 @@ export default class Explorer {
       : this.refreshAllDatabases();
     await useNotebook.getState().refreshNotebooksEnabledStateForAccount();
 
+    //Disable phoenix in case of Vnet or Firewall was enabled.
+    if (!isPublicInternetAccessAllowed()) {
+      useNotebook.getState().setIsPhoenix(false);
```

Comment:
move it to wherever isPhoenix is defined in useNotebooks.ts file..

Alyways - one place for all logic is better.

Reply:
Removed this in current PR